src/App.js

```diff
@@ -1,6 +1,9 @@
 import React, { Component } from "react";
 import axios from "axios";
-import Script from "react-load-script";
+import { Router, Redirect } from "@reach/router";
```

Comment:
Just a note that if we ever wanted to make this a Native app, we'd need to reevaluate which Routing library we use. Not a big deal but good to keep in mind as these web-focused libraries pile on.